regression/jbmc-strings/StringEquals/test_verify.desc

```diff
@@ -0,0 +1,11 @@
+KNOWNBUG
```

Comment:
You still seem to have left this as KNOWNBUG but your comment suggests you think this now works with --throw-runtime-exceptions

Reply:
It doesn't pass on AppVeyor for some reason so I've left it as knownbug, and I will investigate a bit later.